File: sdkwork-app-sdk-typescript/src/types/page-points-exchange-record-vo.ts
import type { PageableObject } from './pageable-object';
import type { PointsExchangeRecordVO } from './points-exchange-record-vo';
import type { SortObject } from './sort-object';

export interface PagePointsExchangeRecordVO {
  totalElements?: number;
  totalPages?: number;
  size?: number;
  content?: PointsExchangeRecordVO[];
  number?: number;
  first?: boolean;
  last?: boolean;
  numberOfElements?: number;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/page_points_exchange_record_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PagePointsExchangeRecordVO:
    total_elements: int = None
    total_pages: int = None
    size: int = None
    content: List[PointsExchangeRecordVO] = None
    number: int = None
    first: bool = None
    last: bool = None
    number_of_elements: int = None
    sort: SortObject = None
    pageable: PageableObject = None
    empty: bool = None
